depthai_common_msgs/CMakeLists.txt

```diff
@@ -35,8 +37,9 @@ generate_messages ()
 ## catkin specific configuration ##
 ###################################
 catkin_package (
-  INCLUDE_DIRS include
+  INCLUDE_DIRS include ${MSGPACK}/include
   CATKIN_DEPENDS message_runtime
+  DEPENDS msgpack-c
```

Comment:
We should add an add_subdirectory in order to remove the warning for missing INCLUDE or LIBRARIES